source/_docs/composer-drupal-8.md

```diff
@@ -9,6 +9,16 @@ contributors:
 
 [Composer](https://getcomposer.org/) is a dependency management tool that allows PHP code to be more easily shared across projects. As Drupal 8 was written, many of its subsystems written specifically for Drupal were replaced by more widely used packages. For instance, Drupal's notoriously unwieldy function for making external requests, `drupal_http_request()`, [was replaced](https://www.drupal.org/node/1862446) by [Guzzle](http://guzzlephp.org/). Guzzle is included in Drupal 8 via Composer.
```

Comment:
Let's not use quite so colorful adjectives to describe the poor drupal_http_request() function. Perhaps just noting that it was replaced, without being descriptive about why, would be better.